app/views/account/InvoicesView.coffee

```diff
@@ -62,12 +66,14 @@ module.exports = class InvoicesView extends RootView
     jqxhr.done =>
       application.tracker?.trackEvent 'Finished invoice payment',
         amount: @amount
+        currency: @currency # added 2.17.2016
         description: @description
 
       # Show success UI
       @state = 'invoice_paid'
       @stateMessage = "$#{(@amount / 100).toFixed(2)} " + $.t('account_invoices.success')
       @amount = 0
+      @currency = '' # added 2.17.2016
```

Comment:
Don't set this to an invalid value after payment completes.  Just leave it as is.